doc/source/io.rst

```diff
@@ -50,6 +50,12 @@ Writing Spatial Data
 
 GeoDataFrames can be exported to many different standard formats using the ``GeoDataFrame.to_file()`` method. For a full list of supported formats, type ``import fiona; fiona.supported_drivers``.
 
+.. note::
+
+    GeoDataFrame can contain more field types than most of the file types. For example tuples or lists
+    can be easily stored in the GeoDataFrame, but saving them to GeoPackage will raise a ValueError.
```

Comment:
```suggestion
    can be easily stored in the GeoDataFrame, but saving them to e.g. GeoPackage or Shapefile will raise a ValueError.
```

(since this is a general note?)